assets/src/adaptivity/scripting.ts

```diff
@@ -161,6 +166,10 @@ export const getAssignStatements = (state: Record<string, any>): string[] => {
   return letStatements;
 };
 
+export const getAssignScriptArray = (state: Record<string, any>): string | string[] => {
```

Comment:
Added this new function else I had to make changes on multiple locations including the unit test cases.

Reply:
why not just call `getAssignStatements` directly?